cmd/podman/shared/intermediate_varlink.go

```diff
@@ -0,0 +1,395 @@
+	m["init-path"] = newCRStringFromVarlink(opts.InitPath)
+	m["interactive"] = newCRBoolFromVarlink(opts.Interactive)
+	m["ip"] = newCRStringFromVarlink(opts.Ip)
+	m["ipc"] = newCRStringFromVarlink(opts.Ipc)
+	m["kernel-memory"] = newCRStringFromVarlink(opts.KernelMemory)
+	m["label"] = newCRStringArrayFromVarlink(opts.Label)
+	m["label-file"] = newCRStringSliceFromVarlink(opts.LabelFile)
+	m["log-driver"] = newCRStringFromVarlink(opts.LogDriver)
+	m["log-opt"] = newCRStringSliceFromVarlink(opts.LogOpt)
+	m["mac-address"] = newCRStringFromVarlink(opts.MacAddress)
+	m["memory"] = newCRStringFromVarlink(opts.Memory)
+	m["memory-reservation"] = newCRStringFromVarlink(opts.MemoryReservation)
+	m["memory-swap"] = newCRStringFromVarlink(opts.MemorySwap)
+	m["memory-swappiness"] = newCRInt64FromVarlink(opts.MemorySwappiness)
+	m["name"] = newCRStringFromVarlink(opts.Name)
+	m["net"] = newCRStringFromVarlink(opts.Net)
+	m["network"] = newCRStringFromVarlink(opts.Network)
+	m["no-hosts"] = newCRBoolFromVarlink(opts.NoHosts)
+	m["oom-kill-disable"] = newCRBoolFromVarlink(opts.OomKillDisable)
+	m["oom-score-adj"] = newCRIntFromVarlink(opts.OomScoreAdj)
+	m["pid"] = newCRStringFromVarlink(opts.Pid)
+	m["pids-limit"] = newCRInt64FromVarlink(opts.PidsLimit)
+	m["pod"] = newCRStringFromVarlink(opts.Pod)
+	m["privileged"] = newCRBoolFromVarlink(opts.Privileged)
+	m["publish"] = newCRStringSliceFromVarlink(opts.Publish)
+	m["publish-all"] = newCRBoolFromVarlink(opts.PublishAll)
+	m["quiet"] = newCRBoolFromVarlink(opts.Quiet)
+	m["read-only"] = newCRBoolFromVarlink(opts.Readonly)
+	m["restart"] = newCRStringFromVarlink(opts.Restart)
+	m["rm"] = newCRBoolFromVarlink(opts.Rm)
+	m["rootfs"] = newCRBoolFromVarlink(opts.Rootfs)
+	m["security-opt"] = newCRStringArrayFromVarlink(opts.SecurityOpt)
+	m["shm-size"] = newCRStringFromVarlink(opts.ShmSize)
+	m["stop-signal"] = newCRStringFromVarlink(opts.StopSignal)
+	m["stop-timeout"] = newCRUintFromVarlink(opts.StopTimeout)
+	m["storage-opt"] = newCRStringSliceFromVarlink(opts.StorageOpt)
+	m["subgidname"] = newCRStringFromVarlink(opts.Subgidname)
+	m["subuidname"] = newCRStringFromVarlink(opts.Subuidname)
+	m["sysctl"] = newCRStringSliceFromVarlink(opts.Sysctl)
+	m["systemd"] = newCRBoolFromVarlink(opts.Systemd)
+	m["tmpfs"] = newCRStringSliceFromVarlink(opts.Tmpfs)
+	m["tty"] = newCRBoolFromVarlink(opts.Tty)
+	m["uidmap"] = newCRStringSliceFromVarlink(opts.Uidmap)
+	m["ulimit"] = newCRStringSliceFromVarlink(opts.Ulimit)
+	m["user"] = newCRStringFromVarlink(opts.User)
+	m["userns"] = newCRStringFromVarlink(opts.Userns)
+	m["uts"] = newCRStringFromVarlink(opts.Uts)
+	m["mount"] = newCRStringArrayFromVarlink(opts.Mount)
+	m["volume"] = newCRStringArrayFromVarlink(opts.Volume)
+	m["volumes-from"] = newCRStringSliceFromVarlink(opts.VolumesFrom)
+	m["workdir"] = newCRStringFromVarlink(opts.WorkDir)
+	// global flag
+	m["trace"] = newCRBoolFromVarlink(opts.Trace)
+	m["syslog"] = newCRBoolFromVarlink(opts.Syslog)
```

Comment:
I don't think we should pass this flag.  This should only tell the client to syslog not the server.  The server is controlled by the varlink command.

Bottom line No passing of global options at all.